src/Microsoft.ComponentDetection.Contracts/BcdeModels/DependencyScope.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ComponentDetection.Contracts.BcdeModels
+{
+    /// <summary>Used to communicate Dependency Scope of Component. 
+    /// Currently only populated for Maven component. 
+    /// The values are ordered in terms of priority, which is used to resolve the scope for duplicate component while merging them.
+    /// </summary>
+    public enum DependencyScope
+    {
+        /// <summary>default scope. dependencies are available in the project during all build tasks. propogated to dependent projects. </summary>
+        Compile = 0,
```

Comment:
Was this the final outcome that DependencyScope shouldn't have the type prefix before their names? i.e. MavenCompile

Reply:
added 'Maven' prefix to every scope. Also based, 'isDevelopmentDependency` on the scope rather than parsing the scope twice.